lecture20240429.tex

```diff
@@ -0,0 +1,58 @@
+\chapter{Lecture 20240429}
```

Comment:
I don't know what lecture number this is, I just know the date. Maybe we can resolve this later.